modules/Virtualizer_module/src/VirtualizerModule.cpp

```diff
@@ -216,6 +216,11 @@ void VirtualizerModule::resetPlayerOrientation()
 {
     std::lock_guard<std::mutex> guard(m_mutex);
     m_cvirtDeviceID->ResetPlayerOrientation();
+
+    oldPlayerYaw = (double)(m_cvirtDeviceID->GetPlayerOrientation());
+    oldPlayerYaw *= 360.0f;
+    oldPlayerYaw = oldPlayerYaw * M_PI / 180;
+    oldPlayerYaw = Angles::normalizeAngle(oldPlayerYaw);
```

Comment:
It is not due to this PR, but I was quite confused that `oldPlayerYaw` do not have the `m_` prefix as the other attributes in the class.

Reply:
Done in 46b10b9